packages/components/src/MultiSelect/_docs/MultiSelect.stories.tsx

```diff
@@ -25,6 +25,8 @@ const meta = {
   },
   args: {
     label: "Breakfast order",
+    description:
```

Comment:
Given this is an optional prop, this should have its own story and docs description instead of being part of the default.